src/effects/VST3/VST3Wrapper.cpp

```diff
@@ -240,12 +240,9 @@ class ComponentHandler : public Steinberg::Vst::IComponentHandler
          ParameterInfo info {};
          mWrapper.mEditController->getParameterInfo(i, info);
 
-         if (info.flags & (ParameterInfo::kIsHidden | ParameterInfo::kIsProgramChange))
+         if ((info.flags & ParameterInfo::kIsReadOnly) != 0)
             continue;
-
-         if ((info.flags & ParameterInfo::kCanAutomate)  == 0)
-            continue;
-
+         
```

Comment:
It's not obvious locally in the code here, but I understand a consequence of this is that changes of factory preset will "dirty" the project so you are prompted to save it when you close.  Am I right?  Is there another reason for this?

Reply:
It is correct, though formally there could be other types of parameter that aren't automatable or hidden. For such parameters it is correct to "dirty" the project too I think.